### ISSUE TITLE
Takeover Request

### ISSUE THREAD

**Issue by Whizfactor** [thclark/wagtail_events#25]
I did like to take over this project. It hasn't received updates for years. I have been trying to get started with a Wagtail package of my own and I think this is one to start with. I need it for Management System i trying to build but i need way more features than it provides and it needs to be updated to match the latest wagtail and other necessary things.

Do consider this proposal and its urgent. @thclark do respond early.

**Comment by thclark** [comment]
Yes! I hate to see a library go to waste and I never use wagtail these days. Reach out to me at tom+wagtail-events@octue.com so we can organise a call to either do a transfer or a simple addition as a collaborator if you prefer.
